src/client/components/responsive-image/responsive-image.vue

```diff
@@ -1,18 +1,101 @@
 <template>
-  <figure>
-    <img :alt="image.alt" :src="image.url" />
+  <figure class="responsive-image" :style="{ maxHeight: `${image.height}px`, maxWidth: `${image.width}px` }">
+    <div class="responsive-image__wrapper" :style="{ paddingBottom: `${ratio}%` }">
+      <picture class="responsive-image__content">
+        <!--[if IE 9]><video style="display: none;"><![endif]-->
+        <source type="image/webp" media="(min-width: 360px)" :srcset="`${imageSource}?fm=webp`">
+        <source type="image/webp" :srcset="`${imageSource}?w360&fm=webp`">
+        <source media="(min-width: 360px)" :srcset="imageSource">
+        <source :srcset="`${imageSource}?w360`">
+        <!--[if IE 9]></video><![endif]-->
+        <img class="responsive-image__image" :alt="image.alt" :src="imageSource" />
+      </picture>
```

Comment:
I think the only thing missing is a `<noscript><img ... /></noscript>` as thanks to Nuxt generate our pages also work without JavaScript. However thanks to the lazy loading strategy, you now don't see any images without JavaScript.